_includes/join-us/join-us-grow-with-strimzi-band.html

```diff
@@ -0,0 +1,35 @@
+<div class="join-us-band">
+  <div class="component-slim grid-wrapper">
+    <div class="width-12-12">
+      <h2 class="blue">Grow with Strimzi</h2>
+      
+      <p>Everyone can join the Strimzi community and grow with it. 
+        It doesn’t matter what your skills are because you can contribute in many different ways. 
+        It also doesn’t matter whether you have just a few minutes to say `Hi` or you can spend several days coding a complicated feature.</p>
+
+      <p>When you contribute to Strimzi - in one of the ways described above - you will become a Strimzi contributor. 
+        We welcome all kinds of contributors - regardless whether you are implementing a feature needed by your company or whether you are just looking for something to spend your evenings with.</p>
+
+      <p>As you contribute more and more, you might focus on some particular areas and become what we call a <a href="https://github.com/strimzi/governance/blob/main/GOVERNANCE.md#component-owners"><i>“component owner”</i></a>. 
```

Comment:
I know this page is fairly short and I haven't seen it with the proper formatting, but I wonder if two headings for "Component Owners" and "Maintainers" would just help to orient people a bit. I think all the information in this page is good, but when I was previewing it in a browser it seemed like quite a wall of text.